Makefile

```diff
@@ -100,6 +100,7 @@ test: test-unit
 test-unit: netavark-testplugin
 	go test --tags $(BUILDTAGS) -v ./libimage/...
 	go test --tags $(BUILDTAGS) -v ./libnetwork/...
+	go test --tags $(BUILDTAGS) -v ./internal/...
```

Comment:
Maybe it is time to change this to `go test --tags $(BUILDTAGS) -v ./...`, people will forget to add the packages when they add top level dirs.

Reply:
Agreed.  I opened https://github.com/containers/common/issues/1681 to not forget about it.